int8 numImages
---
sensor_msgs/Image[] images